package.json

```diff
@@ -13,7 +13,9 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
-    "openapi:bundle": "bundle-api-spec -i openapi/index.yaml.template -o openapi/index.yaml -V $npm_package_version",
+    "openapi:bundle": "npm-run-all openapi:bundle:*",
+    "openapi:bundle:internal": "bundle-api-spec -i openapi/index_internal.yaml.template -o openapi/index.yaml -V $npm_package_version",
```

Comment:
Do we want to keep `index.yaml` as the name of the output file? Or maybe we can name it `index_internal.yaml` just like the other one

Reply:
I did this to avoid breaking any api that could be pointing it.